backend/hackathons/controllers/platforms/devpostController.js

```
@@ -88,37 +87,37 @@ async function fetchPageWise (url, page) {
+      for (let pageNumber = 2; pageNumber <= Math.ceil(totalHackathons / per_page); ++pageNumber) {
         const hackathons = await fetchPageWise(`${url}&page=${pageNumber}`, pageNumber);
         listOfHackathons.push(...hackathons);
       }
 
       resolve(listOfHackathons);
-    }catch(error){
+    } catch (error) {
       console.log(error);
       resolve([]);
     }
   });
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)